control-plane/agents/src/bin/ha/cluster/switchover.rs

```diff
@@ -235,6 +266,12 @@ pub struct SwitchOverEngine {
 
 /// Number of Switchover worker tasks to be spawned.
 const WORKER_NUM: u8 = 4;
+/// Sleep duration during Fast requeue phase.
+const FAST_REQUEUE: u64 = 10;
+/// Sleep duration during Slow requeue phase.
+const SLOW_REQUEUE: u64 = 30;
+/// Number of Fast request requeue per stage.
+const NUM_OF_FAST_REQUEUE: u64 = 18;
```

Comment:
You can use these as an enum, eg:
```rust
enum ReQueue {
    /// Sleep duration during Fast requeue phase.
    Fast = 10,
    /// Sleep duration during Slow requeue phase.
    Slow = 30,
    /// Number of Fast request requeue per stage.
    NumFast = 18
}
// and then just use them like
ReQueue::Fast as u64/whatever
```

Reply:
Have you thought about using a proper backoff algorithm? There's a rust crate for that.
Also is 30 "slow" enough for slow?

Reply:
Post 3 minutes we requeue after 30 secs. Do you have any suggestion?

Reply:
I don't know, 1 minute? Not sure.. What do you think @Abhinandan-Purkait ?

Reply:
Ill keep it as is for now and merge PR. We can always change it when we decide.